Releases.md

```diff
@@ -6,6 +6,32 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at:
 https://github.com/denoland/deno_install
 
+### 1.23.1 / 2022.06.23
+
+- BREAKING(ext/ffi): Remove `Deno.UnsafePointer` indirection (#14915)
+- feat(ext/ffi): Callbacks (#14663)
```

Comment:
To make it more clear, maybe change this to `feat(unstable/ffi)` and above to `BREAKING(unstable/ffi)`.